### ISSUE TITLE
Translations from qml files got lost

### ISSUE THREAD

**Issue by Jojo-Schmitz** [Jojo-Schmitz/MuseScore#536]
All translations from qml files, those listed in mscore/qml.qrc, are lost
In Mu3.6.2 they show in the .ts files as e.g.:
```xml
<context>
    <name>PalettesWidgetHeader</name>
    <message>
        <location filename="../../mscore/qml/palettes/PalettesWidgetHeader.qml" line="75"/>
        <source>Add Palettes</source>
        <translation>Palette hinzufügen</translation>
    </message>
```
in 3.7 they show as:
```xml
<context>
    <name>PalettesWidgetHeader</name>
    <message>
        <source>Add Palettes</source>
        <translation type="vanished">Paletten hinzufügen</translation>
    </message>
 ```
i.e. no `location` tag and an added `type="vanished"` property for the `translation` tag.

I wonder whether this is either because of using a different Qt version than 3.6.2 or because I updated them (`lupdate`) on Windows (which is known to take ages, and therefor used get created on Mac, but I don't have one...)

**Comment by Jojo-Schmitz** [comment]
Manually adding back the `location` tag and removing the  `type="vanished"` property for the `translation` tag fixes it, but would be a royal PITA to do for all missing strings in all languages

**Comment by Jojo-Schmitz** [comment]
Seems I found a way to fix it, at least for Windows, see #537
